dev-requirements.txt

```diff
@@ -1,3 +1,19 @@
+astroid==2.2.5
```

Comment:
True. This file looks very similar to the production requirements, however, it contains extra dependencies that are needed for development. I saw this is a nice approach since it means when we clone this repo it would only require one `pip install` run for the dev dependencies instead of two files.
This was in line with @Kyppy 's question during the kick-off call. This will help in removing the need to cherry-pick dependencies except only when pushing to production. 
What is your opinion on this? Does this sound like a good plan?

Reply:
I understand the problem you are trying to resolve and while doing more research on it, I've come across a possible solution. Instead of including all the dependencies in the requirements.txt, we could include `-r requirements.txt` in the requirements-dev.txt. This way the dependencies in the requirements.txt will also be installed. I found this solution here
https://stackoverflow.com/questions/17803829/how-to-customize-a-requirements-txt-for-multiple-environments
Do you think we could look more into this solution?

Reply:
I think that is actually a really clever way of doing it. I will refactor.